### ISSUE TITLE
Can ext. storages replace load shedding?

### ISSUE THREAD

**Issue by lukasoldi** [openego/eTraGo#90]
...they should in theory. But there are cases where this does not work and one receives infeasible results, esp. with the full model for DE.

@kimvk solved this for the extendable lines with this: https://github.com/openego/eTraGo/blob/c4e2cd2c273915b746e2ab02b7f3137e656e4e2c/etrago/appl.py#L111-L117

**Comment by ulfmueller** [comment]
I tried out a similiar approach for the ext. storages today. But it did not help...

**Comment by lukasoldi** [comment]
Could you share your preferences and what exactly you applied?

**Comment by ulfmueller** [comment]
I tried it out very quickly on the server and with my limited hardware right now I cannot access it. But I will try straight from my mind:
[Here](https://github.com/openego/eTraGo/blob/dev/etrago/appl.py#L241) I added two lines like this:

network.storage_units.s_nom_min = 0 
network.storage_units.s_nom_max = float ("+inf") 

and then I calculated I think from 2305 to 2328 with storage extendable on and load shedding off. No clustering.gurobi.gridversion 0.2.11. and probably a random noise.

**Comment by ulfmueller** [comment]
Again I forgot about one mechanism. A plausible reason would be that the time span is too short in order to be able to fill up the storage... The influence of the filling level is probably the reason.

**Comment by lukasoldi** [comment]
true...but lets keep the ticket open in order to keep that point up for validation.

**Comment by wolfbunke** [comment]
@BartelsJ did you add this to our validation list and can we close this issue? @ulfmueller
